src/redteam/commands/generate.ts

```diff
@@ -29,8 +29,8 @@ import { resolveConfigs } from '../../util/config/load';
 import { writePromptfooConfig } from '../../util/config/writer';
 import { getCustomPolicies } from '../../util/generation';
 import { printBorder, setupEnv } from '../../util/index';
-import { promptfooCommand } from '../../util/promptfooCommand';
 import invariant from '../../util/invariant';
+import { promptfooCommand } from '../../util/promptfooCommand';
```

Comment:
can you fix whatever is causing these file changes?  Happy to change the import linter rule to make this better. You also modified src/commands/eval.ts above